flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableSinkITCase.java

```diff
@@ -496,6 +496,48 @@ public void testWritingNoDataToPartition() throws Exception {
         assertBatch("target_table", Arrays.asList());
     }
 
+    @Test
+    public void testWriteSuccessFile() throws Exception {
```

Comment:
Also, could you please add tests to verify the following two cases:
1: disable wrtiting success file 
2: change success file name